src/mscorlib/model.xml

```diff
@@ -12588,7 +12588,7 @@
       <Member Name="get_TraceLevel" />
     </Type>
     <!--End of PFX types -->
-    <Type Name="System.Console">
+    <Type Name="System.LowLevelConsole">
```

Comment:
Where do we still use this in System.Private.CoreLib?  Is it just for debugging?